python/pip_package/setup.py

```diff
@@ -0,0 +1,103 @@
+# pylint: disable=invalid-name, exec-used
+"""Setup TVM package."""
+from __future__ import absolute_import
+import os
+import shutil
+import sys
+import platform
```

Comment:
I feel maybe we could directly move this folder to .. and make the setup.py support normal setup as well as dist build